### DISCUSSION TITLE
Release weight

### DISCUSSION THREAD

**Discussion by RicoSuaveGuapo** [xN1ckuz/Crosswalks-Detection-using-YOLO#3] [General]
Thank you for your hard work on the crosswalk dataset and model training~
For personal research, is there any chance to release your trained model for experiment comparison?

**Comment by xN1ckuz** [comment]
Hello. In the Release section there is the v.14 exported in ONNX format. You're free to use if you want

**Reply by RicoSuaveGuapo** [reply]
Thanks!
